app/src/main/java/tech/ula/utils/FilesystemUtility.kt

```diff
@@ -87,4 +100,74 @@ class FilesystemUtility(
             }
         }
     }
+
+    fun backupFilesystemByLocation(execDirectory: String, directoryName: String, backupName: String, destinationDir: File) {
+        val backupLocation = "../${backupName}"
```

Comment:
This work should be moved into a script and placed in our [Support Repository](https://github.com/CypherpunkArmory/UserLAnd-Assets-Support) to keep bash clutter down in-app and to keep logical grouping maintained.

Same with  restoreFilesystemByLocation.